accepted/0000-remove-dependency-on-fulcio.md

```diff
@@ -0,0 +1,69 @@
+# RFC for linking packages to their source and build without using Sigstore Fulcio
+
+## Summary
+
+As it is showed in a recent paper that Sigstore Fulcio is an unnecessary trusted third party for linking npm packages to their source and build, we suggest the npm team consider the new approach presented in that paper which can be found at:
+
+https://eprint.iacr.org/2023/003
+
+The new approach not only eliminates the unnecessary dependency on an external black-box CA but also accelerates the signing process.
```

Comment:
What is meant by "external black-box CA"?

Fulcio uses certificate transparency, like other webPKI CAs, to allow users to monitor certificate issuances. Verifiers should only trust certificates that have been publicly logged, much like verifiers should only trust artifacts logged in Rekor. This makes misconfigured or malicious CA actions publicly auditable.

Reply:
From the perspective of a theoretical cryptographer, a CA is a black box with or without certificate transparency. Sorry if the term means different to you.

Reply:
Any server component that cannot be publicly audited will be a black box. How don’t you know that the identity provider isn’t issuing tokens with your identity to malicious actors? Without a transparency mechanism, this behavior can’t be audited.